### PR TITLE
feat(prebuilt): add phpmyadmin

### PR DESCRIPTION
**Pull request by yuaanlin**

### CODE REVIEW

File: prebuilts/phpmyadmin.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[source]
+image = "phpmyadmin/phpmyadmin"
+
+[[ports]]
+id = "web"
+port = 80
+type = "HTTP"
+
+[env]
+PMA_HOST = { default = "mysql.zeabur.internal" }
```
Comment (pan93412) [inline prebuilts/phpmyadmin.toml:17]:
```suggestion
PMA_HOST = { default = "${MYSQL_HOST}" }
```

File: prebuilts/phpmyadmin.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PMA_PORT = { default = "3306" }
```
Comment (pan93412) [inline prebuilts/phpmyadmin.toml:18]:
```suggestion
PMA_PORT = { default = "${MYSQL_PORT}" }
```

File: prebuilts/phpmyadmin.toml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+PMA_USER = { default = "root" }
```
Comment (pan93412) [inline prebuilts/phpmyadmin.toml:19]:
```suggestion
PMA_USER = { default = "${MYSQL_USERNAME}" }
```

File: prebuilts/phpmyadmin.toml (hunk 1 above)
Comment (yuaanlin) [reply prebuilts/phpmyadmin.toml]: https://github.com/zeabur/zeabur/pull/380/commits/4bc875077776ffe08ae0c5ed8e16aff78efdd325

File: prebuilts/phpmyadmin.toml (hunk 2 above)
Comment (yuaanlin) [reply prebuilts/phpmyadmin.toml]: https://github.com/zeabur/zeabur/pull/380/commits/4bc875077776ffe08ae0c5ed8e16aff78efdd325

File: prebuilts/phpmyadmin.toml (hunk 3 above)
Comment (yuaanlin) [reply prebuilts/phpmyadmin.toml]: https://github.com/zeabur/zeabur/pull/380/commits/4bc875077776ffe08ae0c5ed8e16aff78efdd325

**Review by pan93412 — APPROVED** [review]

**Pull request merged**